nss-day-cohort-41/react-nutshell-thepecans
# Add first event style minimum requirements

# Description
Created logic to eliminate past events from display. Separated the soonest upcoming event from the array, and passed logic to the EventCard to further style that first event. Applied minimum requirements: larger, bold text, non-white background.

The event list still looks awful overall. Further styling will be applied later to clean it up and make it pretty.

Fixes #5 (partial)
## Type of change

# Testing Instructions
Navigate to the ".../events" page. Make sure there are one or more events in your database. If not, create some events. The soonest event will appear at the top of the list with the aforementioned style changes.

# Checklist:
